modules/json_content/src/Controller/FrontController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Drupal\json_content\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+
+class FrontController extends ControllerBase {
+
+  /**
+   * Displays the login form on the homepage and redirects authenticated users.
+   */
+  public function frontpage() {
+    $build = [];
+    if (\Drupal::currentUser()->isAnonymous()) {
+      $build['heading'] = [
+        '#type' => 'markup',
+        '#markup' => $this->t('Please login for access to the content repository.'),
+      ];
+      $build['form'] = $this->formBuilder()
```

Comment:
This kind of thing can get dicey. How about if we just set a message and redirect to /user/login instead?